tests/sanity/test/e2e/test-01-login.js

```diff
@@ -290,7 +292,14 @@ describe('test MVD login page', function() {
 
     // check popup menu items
     const menuItems = await getElements(popup, '.launch-widget-row > .app-label');
-    expect(menuItems).to.be.an('array').that.have.lengthOf(PRE_INSTALLED_APPS.length);
+
+    if (dockerTest) {
+      // Docker scenario will not have IP Explorer, thus installed apps should be one less the total count
+      expect(menuItems).to.be.an('array').that.have.lengthOf(PRE_INSTALLED_APPS.length-1);
```

Comment:
Could we define another `constant PRE_INSTALLED_APPS_DISABLED_FOR_DOCKER = ['IP Explorer'];`, then here we can say  `expect(menuItems).to.be.an('array').that.have.lengthOf(PRE_INSTALLED_APPS.length-PRE_INSTALLED_APPS_DISABLED_FOR_DOCKER.length)`